src/components/CreateEventForm/index.jsx

```diff
@@ -0,0 +1,165 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Form, Input, Button, Select } from 'antd';
+
+import './style.less';
+
+const { Option } = Select;
+const { TextArea } = Input;
+
+const months = [
+  'January',
+  'February',
+  'March',
+  'April',
+  'May',
+  'June',
+  'July',
+  'August',
+  'September',
+  'October',
+  'November',
+  'December',
+];
```

Comment:
You could also generate a list of months through this https://www.w3schools.com/jsref/jsref_getmonth.asp